test/main/contracts/sources/etherscan.test.js

```diff
@@ -95,7 +95,7 @@ describe('#fetchEtherscanContract', () => {
     mockEtherscanApi(400)
 
     return expect(fetchEtherscanContract(contractAddress, 1)).resolves.toBeUndefined()
-  })
+  }, 500)
```

Comment:
did we have some failures here? just ran this locally and none of the tests took more than 20 ms so just curious what happened

Reply:
My machine is probably running slow but these tests consistently fail for me to the extent where I have to disable husky in order to push anything.